resourcegraph/Import-ResourceGraphToLogAnalytics.ps1

```diff
@@ -0,0 +1,92 @@
+#Requires -Modules Az.Accounts, Az.ResourceGraph
+
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$Query,
+    [Parameter(Mandatory = $true)]
+    [string]$ManagementGroupIdToCheck,
+    [Parameter(Mandatory = $true)]
+    [string]$LogType, # Specify the name of the record type that you'll be creating
+    [Parameter(Mandatory = $true)]
+    [string]$CustomerId  # Replace with your Workspace ID
+)
+
+# Run Resource Graph
+
+$ErrorActionPreference = "Stop"
+
+$sharedKey = Get-AutomationVariable law_sharedkey
+$default_subscription  = Get-AutomationVariable default_subscription
+
+try{
+	Connect-AzAccount -Identity -Subscription $default_subscription | Out-Null
```

Comment:
Is it necessary to add a subscription :O usually -Identity is enough isn't it?

Reply:
@QBY-MarkusMaring